bigframes/ml/llm.py

```diff
@@ -421,6 +421,11 @@ def to_gbq(
 class GeminiTextGenerator(base.BaseEstimator):
     """Gemini text generator LLM model.
 
+    Note: This product or feature is subject to the "Pre-GA Offerings Terms" in the General Service Terms section of the
```

Comment:
We should follow a docstring format like this for note:
```
.. note::
    This product or feature is subject to the "Pre-GA Offerings Terms" in the General Service Terms section of the
```

Reply:
Done